app/modules/assets/models.py

```diff
@@ -129,8 +123,7 @@ def get_symlink(self):
     def get_derived_path(self):
         asset_group_abspath = self.asset_group.get_absolute_path()
         assets_path = os.path.join(asset_group_abspath, '_assets')
-        asset_symlink_filepath = os.path.join(assets_path, 'derived', self.get_filename())
-        return asset_symlink_filepath
+        return os.path.join(assets_path, 'derived')
```

Comment:
isnt this missing the part where it adds `self.get_filename()` onto the whole thing, or am i missing something?

Reply:
Yeah, `get_derived_path` used to return something like `/data/var/asset_group/d072167c-992b-4bff-8548-d9044d0ea792/_assets/derived/c48acd4e-0042-434b-b130-fe9fb6812118.jpg` and then we strip the file extension and add the format.  I changed `get_derived_path` to return the derived directory `/data/var/asset_group/d072167c-992b-4bff-8548-d9044d0ea792/_assets/derived/` and then add `self.guid`, `format` and file extension.

Reply:
this makes sense.  not sure if there will ever be a need for a method that returns the whole thing + filename; but we can cross that bridge when we come to it!   thanks